lib/manager/components/HomeProjectDropdown.js

```diff
@@ -17,20 +18,11 @@ type Props = {
   visibleProjects: Array<Project>
 }
 
-type State = {
-  selectedOption: any
-}
```

Comment:
Generally we try not to introduce local component state unless it's really warranted. Here, we're already tracking the selectedOption via the `activeProject`, which comes from the component's props (and ultimately from the URL that we update in handleChange).